### ISSUE TITLE
What is the nature of matter  at extreme densities? (proposed key question)

### ISSUE THREAD

**Issue by ethrane** [ethrane/transients#3]
@ethrane and @kauchettl propose this as a key question. This question would cover all manner of things related to neutron stars. We expect some synergy with Group 1.2 (Stars, Planets, & the Galaxy). This question is similar to one that was asked in the last decadal plan, "What is the nature of matter and gravity at extreme densities?" However, we like the idea of moving "gravity" into a separate key question; see Issue https://github.com/ethrane/transients/issues/2.

**Comment by SimonStevenson** [comment]
Individual questions for neutron stars and black holes may be too niche/restrictive (and what about white dwarfs). Perhaps consider a broader question on stellar remnants, such as "What happens after stars die?"

**Comment by jeffcooke** [comment]
Agree. We address extreme densities, what about extreme energies? This is where transients shine and are nature's laboratories for this. These are energies that cannot be created on Earth.

**Comment by plasky** [comment]
I support this question as written (note that I would even argue it does not include black holes!). This is one of the few areas of astrophysics where people working on fundamental particle physics really care about the input that astronomers and astrophysicists have. They can calculate from first principles how matter should behave orders of magnitude _more_ dense than in neutron stars, and how it behaves at densities order of magnitudes _less_. But they can't calculate how it should behave in between those densities, which leaves large gaps in understanding the standard model of particle physics. For this reason I am personally against trying to broaden this question to the point where the particle physicists would stop caring. Note that they do not care about e.g., white dwarfs; you can calculate from first principles how matter should behave there, which is why I would not classify it as "extreme".

**Comment by cwjames1983** [comment]
@plasky I agree it shouldn't be too broad: but this doesn't have to mean particle physicists care about all the science covered by it. If you look at other categories, any given person cares about perhaps 25% of a question. E.g. I would vote this question higher if it included pulsar/FRB emission physics, which particle physicists don't care about at all. But it's also clearly related to NS.

**Comment by bersavosh** [comment]
@plasky, one can argue the same about black holes (calculating from first principles) and say that all the testing GR to the nth order is not pushing science forward significantly. I would argue in support of considering WDs as a part of this question mainly due to the recent discoveries of pulsating WDs and ultra-long period radio transients. Both classes of object that have only been discovered in the past decade and there is really no easy way to explain their physics properly with our current understanding of WDs (or NSs).

**Comment by ethrane** [comment]
> E.g. I would vote this question higher if it included pulsar/FRB emission physics, which particle physicists don't care about at all. But it's also clearly related to NS.

@cwjames1983, FWIW, I think this would be in the scope of the question as @kauchettl and I envisioned it: all manner of things related to neutron stars.

**Comment by Ch-Wolf** [comment]
❤️

**Comment by bwmeyers** [comment]
Just noting it here as I'm not sure where else to place it, but thinking about pulsars specifically. I think we need to be explicit about Australia's contributions to pulsar astronomy (e.g., discoveries and on-going timing activities) both historically and present/in future  (e.g., pulsar surveys with pre-cursors, like SMART with the MWA, continued support and use of Parkes/Murriyang). Given pulsar's versatility in terms of studying dense matter, gravity, high energy particle creation, ISM on scales inaccessible to any other technique, binary and stellar evolution, it would be a real shame to not continue pushing that as something that Australia's community is good at and well situated to continue as trailblazers. Similarly, from a GW perspective, especially given the splash that happened mid-2023 about nHz SGWB evidence from the IPTA. In that way, pulsar timing array(s) are multi-messenger observatories that can tell us a lot about early galaxy evolution and other "new physics", and (eventually) individual nHz-ish GW sources. 

All this is to say that I couldn't see any mention of these things in any of the issues here, and while I agree that they may also neatly fall into other working groups, it seems important for the transients/time-domain group to also have a spotlight on that, indicating cross-over between multiple communities.

**Comment by ethrane** [comment]
@bwmeyers, the way I see it, pulsar timing is an important ingredient for at least a couple proposed key questions: this one, [the black hole question](https://github.com/ethrane/transients/issues/2), etc. I agree with you that pulsar _discovery_ helps facilitate all this science. I believe we can/should emphasise in the key questions / infrastructure priorities.
